generators/client/templates/angular/src/main/webapp/app/layouts/error/error.route.ts.ejs

```diff
@@ -24,25 +24,17 @@ export const errorRoute: Routes = [
   {
     path: 'error',
     component: ErrorComponent,
-    data: {
-      pageTitle: 'error.title',
-    },
+    title: 'error.title',
   },
   {
     path: 'accessdenied',
     component: ErrorComponent,
-    data: {
-      pageTitle: 'error.title',
-      errorMessage: 'error.http.403',
-    },
+    title: 'error.http.403',
```

Comment:
`errorMessage` is not related to the title:
https://github.com/qmonmert/generator-jhipster/blob/a854f9a13767b510dfc58719ec3527259fed3977/generators/client/templates/angular/src/main/webapp/app/layouts/error/error.component.ts.ejs#L46-L53

```suggestion
    data: {
      errorMessage: 'error.http.403',
    },
    title: 'error.title',
```

You can use the specific title if you want.